public/app/features/dashboard/components/DashNav/DashNav.tsx

```diff
@@ -158,7 +160,7 @@ export const DashNav = React.memo<Props>((props) => {
         <ModalsController key="button-dashboard-comments">
           {({ showModal, hideModal }) => (
             <DashNavButton
-              tooltip="Show dashboard comments"
+              tooltip={t({ id: 'dashboard.toolbar.comments', message: 'Show dashboard comments' })}
```

Comment:
Since most of the other id's which have actions in the messages they refer to, are verbs, maybe it's better to set this one `dashboard.toolbar.comments-show` in case we have `dashboard.toolbar.comments-hide` for hiding and `dashboard.toolbar.comments` for no actions, just a title